docs/nlu/choosing-a-pipeline.rst

```diff
@@ -68,6 +68,25 @@ We do not recommend that you use these; mitie support is likely to be deprecated
     You need to provide enough data for both intents and entities.
 
 
+Class imbalance
+---------------
+
+In order to mitigate class imbalance problem, we introduced ``balanced`` batching strategy for
+``supervised_embeddings`` pipeline. This algorithm ensures that all classes are represented
+at least in subsequent batches, still mimicking the fact that some classes are more frequent than others.
```

Comment:
do you mean at least once _every other_ batch?

Reply:
that depends on the number of classes and batch size it could be once in every 3, but importantly there is a sequence of batches that will have all classes

Reply:
ok. that's quite complicated to explain. we could say  

"ensures that all classes are represented in every batch, or at least in every `n`th batch (with `n` as small as possible)"

Reply:
do you think it simpler?

Reply:
then we'd need to explain what n is

Reply:
what about _in every batch, or at least in as many batches as possible_

Reply:
As many subsequent batches as possible?

Reply:
sounds good